### ISSUE TITLE
[Feature Request] Provide a custom style sheet for the Japanese version of the page.

### ISSUE THREAD

**Issue by ShiMeiWo** [quinton-ashley/p5play-web#31]
I am checking the HTML created by the site generator in my browser, but on Windows 10 Japanese version, the default font of the browser is `Meiryo`, so it looks like the below:
![Meiryoフォント](https://github.com/quinton-ashley/p5play-web/assets/11846146/26f3696c-52a3-4a1d-9efc-206f14013a77)
**(Note that the Colombian flag is not displayed!)**

As a guess, the following presentation may feel better to the person building the site:
![Notoフォント](https://github.com/quinton-ashley/p5play-web/assets/11846146/18a969fe-7bfa-4601-80fc-6c403eef3383)

On the other hand, when I checked the display on the smartphone, the text was legible and the Colombian flag was displayed, but I found that this was because of the use of the [`Noto` font series](https://fonts.google.com/noto) implicitly installed on the device, as shown below:
![スマートフォン](https://github.com/quinton-ashley/p5play-web/assets/11846146/827e8fc5-ebc6-4d9d-b9af-6884c3c96d0a)

Based on the above, I suggest that the Japanese version of the page display characters in `Noto Sans JP` and `Noto Color Emoji`, just as the smartphone version does. These are Web fonts, so they can be used without having to be hosted on your own server.

ex)

```html
<!DOCTYPE html>
<html lang="ja" class="ja">

<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>p5play</title>
	<link rel="icon" href="/assets/p5play_logo.svg">
	<link rel="preconnect" href="https://fonts.googleapis.com">
	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
	<link href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP&family=Noto+Color+Emoji&display=swap" rel="stylesheet">
<!-- snip! --->
```

```css
body {
  font-family: "Noto Sans JP", "Noto Color Emoji", "Yu Gothic", "Hiragino Sans", sans-serif;
}
```

Thanks!

**Comment by quinton-ashley** [comment]
Good idea! I will edit the lang/build.js script to add that font.

**Comment by ShiMeiWo** [comment]
Since you are busy working on the module side, I tried to do some work on my end.

## For h1 and h2 tags, `Mont` font is preferred.

![image](https://github.com/quinton-ashley/p5play-web/assets/11846146/c730d6d8-aceb-4da2-bc88-12784da51a0a)

## Fixed an unwanted scroll gutter is shown in the top navigation area when narrowing a PC browser's width.

![image](https://github.com/quinton-ashley/p5play-web/assets/11846146/939ea1dd-9bb9-48d8-88ed-d6ad3c01a1e1)

## How it looks on the iPads.

![image](https://github.com/quinton-ashley/p5play-web/assets/11846146/ab375794-ebca-4b5f-8ac6-d3c77d471611)

## How it looks on the iPhones.

![image](https://github.com/quinton-ashley/p5play-web/assets/11846146/537f599a-8a34-404f-a962-eae8188a1e5a)

----

I will create a PR for this commit later, so please let me know if there are any undesirable implementations.

Thanks!

**Comment by quinton-ashley** [comment]
fixed by pr #34
